docs/ca/dev/preparation-for-going-live.md

```diff
@@ -84,8 +84,8 @@ Update them if plans change.
+| TES-05 | Testing | Conduct load tests                                     | Test how the platform performs under real-life circumstances. Define the performance acceptance criteria - the sample testing data should be comparable to the size and complexity of the production data. Monitor the application's performance metrics, such as response times or resource usage, during the tests. <br> Inform us about your load and performance test plans via the [Support Portal](https://support.spryker.com/s/) at least 3 days in advance and share the results so that we can fine-tune the environment to meet your specific requirements.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           | Some options to define key metrics: <br>- Historical data: if you have existing traffic data–for example, from Google Analytics <br>- [Industry benchmarks](https://databox.com/website-traffic-benchmarks-by-industry): research typical traffic volumes for similar e-commerce sites in your industry <br>- Business goals: Consider target sales, conversion rates, and marketing campaigns to estimate expected traffic                                 
```

Comment:
@andriitserkovnyi can we make "at least 3 days in advance" more visible somehow?